config.go

```diff
@@ -1129,7 +1115,12 @@ func parseRPCParams(cConfig *chainConfig, nodeConfig interface{}, net chainCode,
 				" %v, cannot start w/o RPC connection",
 				err)
 		}
-		nConf.RPCUser, nConf.RPCPass = rpcUser, rpcPass
+		if nConf.RPCUser == "" {
+			nConf.RPCUser = rpcUser
```

Comment:
I think `extractBtcdRPCParams` might fail if the conf file is not found. We should instead try to 1) extract from conf file and continue whether it succeeds or not, 2) overwrite what we foudn in the conf file with what was specified on the command line, 3) check that everything is set before returning.

Reply:
I think it is the same problem for `bitcoind`. The strategy outlined in my original comment can be used for both.

(sorry for super late reply 🤐 )

Reply:
If you feel motivated to grab this PR go ahead. 

We ended up forking LND in BTCPay for this commit and some other that we need, we rebase our changes on every release.